src/ListOFGoods.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export class ListOfGoods extends React.Component {
+  state = {
+    goods: [...this.props.goods],
+  }
```

Comment:
Why do you add goods to state? You can just use this.props for rendering list

Reply:
Good point